app/views/staff/shared/activities/_table.html.haml

```diff
@@ -0,0 +1,29 @@
+- unless activities.empty?
+  %table.govuk-table.activities
+    %thead.govuk-table__head
+      %tr.govuk-table__row
+        %th.govuk-table__header
+          = t("table.header.activity.identifier")
+        %th.govuk-table__header
+          = t("table.header.activity.title")
+        %th.govuk-table__header
+          = t("table.header.activity.level_a")
+        %th.govuk-table__header
+          = t("table.header.activity.level_b")
+        %th.govuk-table__header
+          = t("table.header.activity.level_c")
+        %th.govuk-table__header
+
+    %tbody.govuk-table__body
+      - activities.each do |activity|
+        %tr.govuk-table__row{ id: activity.id }
+          %td.govuk-table__cell= activity.identifier
+          %td.govuk-table__cell= activity.title
+          %td.govuk-table__cell= activity.parent_activities.first&.title
+          %td.govuk-table__cell= activity.parent_activities.second&.title
+          %td.govuk-table__cell= activity.parent_activities.third&.title
```

Comment:
This seems odd to me, when the activity the row is about is a Level C activity, the "Level C"  column is blank. I suppose the presence of a value in this column is the only indicator that the row contains a "Level D" activity..?